device/iot-device-samples/pnp-device-sample/pom.xml

```diff
@@ -22,8 +22,14 @@
         <gson.version>2.8.6</gson.version>
         <slf4j-version>1.7.25</slf4j-version>
         <maven-jar-plugin-version>3.1.0</maven-jar-plugin-version>
+        <pnp-helper-version>1.0.0</pnp-helper-version>
     </properties>
     <dependencies>
+        <dependency>
+            <groupId>com.microsoft.azure.sdk.iot.pnp</groupId>
+            <artifactId>pnp-helper</artifactId>
+            <version>${pnp-helper-version}</version>
```

Comment:
Is this defined somewhere? I see that it was removed from the root pom.

Reply:
https://github.com/Azure/azure-iot-sdk-java/pull/866/files#diff-25a676cbb100784e8babd375ba346450R25